src/main/java/ILanguage.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *   Copyright (c) 2020 Jalasoft.
+ *
+ *   This software is the confidential and proprietary information of Jalasoft.
+ *   ("Confidential Information"). You shall not disclose such Confidential
+ *   Information and shall use it only in accordance with the terms of the
+ *   license agreement you entered into with Jalasoft.
+ */
+
+/**
+ * This interface defines the type of object for the classes that implement it will use
+ */
+public interface ILanguage {
+    /**
+     * @param params whit this object this method will receive its parameters
```

Comment:
@carlosmc23 Please check spelling and explain better this part

Reply:
This variable called params will be contain all parameter abut the file that we will use to complile